AppHandling/Run-AlPipeline.ps1

```diff
@@ -13,6 +13,8 @@
   License file to use for AL Pipeline.
  .Parameter containerName
   This is the containerName going to be used for the build/test container. If not specified, the container name will be the pipeline name followed by -bld.
+ .Parameter generateErrorLog
+  Switch parameter on whether to generate an alerts log file. Default is false.

```

Comment:
Add to the comment that an alerts log file is generated for each compiled app file and placed in the output folder next to the app file.
(I think that is what it does right?)